public/app/features/dashboard-scene/saving/DashboardSceneChangeTracker.ts

```diff
@@ -42,96 +25,25 @@ export class DashboardSceneChangeTracker {
       return;
     }
 
-    // Any change in the panel should trigger a change detection
-    // The VizPanelManager includes configuration for the panel like repeat
-    // The PanelTimeRange includes the overrides configuration
+    // if (payload.changedObject instanceof SceneTimeRange) {
+    //   return this.runDirtyCheck();
+    // }
```

Comment:
not sure if a simple time range change should mark dashboard as dirty, but open to debate